package.json

```diff
@@ -18,5 +18,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "prop-types": "^15.8.1"
   }
```

Comment:
This... shouldn't work ? The package should be installed in the `frontend` package.json, not in the global one. I'm surprised, I didn't think it'd fetch the package one level higher